Description: Mono bindings are installed to a non-multi-arch location.
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/bindings/mono/ayatana-appindicator-sharp-0.1.pc.in
+++ b/bindings/mono/ayatana-appindicator-sharp-0.1.pc.in
@@ -1,6 +1,6 @@
-prefix=@prefix@
+prefix=/usr
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=/usr/lib
 assemblies_dir=${prefix}/lib/cli/ayatana-appindicator-sharp-0.1
 
 Name: ayatana-appindicator-sharp
